code/game/machinery/requests_console.dm

```diff
@@ -363,7 +363,7 @@ GLOBAL_LIST_EMPTY(allRequestConsoles)
 	set_light(2)
 
 /obj/machinery/requests_console/proc/remind_unread_messages()
-	if(newmessagepriority == RQ_NONEW_MESSAGES)
+	if(newmessagepriority == RQ_NONEW_MESSAGES && reminder_timer_id != TIMER_ID_NULL)
```

Comment:
This will now make the machine say the text below if the timer doesnt exist. Best to split the statements and only del the timer if needed, but still return